src/Common/ClipboardButton/ClipboardButton.tsx

```diff
@@ -65,28 +66,22 @@ export default function ClipboardButton({
         }
     }, [trigger, handleCopyContent])
     return (
-        <div className="icon-dim-16 flex center">
-            <Tippy
-                className="default-tt"
-                content={copied ? copiedTippyText : 'Copy'}
-                placement="bottom"
-                visible={copied || enableTippy}
-                arrow={false}
+        <Tippy
+            className="default-tt"
+            content={copied ? copiedTippyText : 'Copy'}
+            placement="bottom"
+            visible={copied || enableTippy}
+            arrow={false}
+        >
+            <button
+                type="button"
+                className={`dc__outline-none-imp p-0 flex bcn-0 dc__no-border ${rootClassName}`}
```

Comment:
Can we replace bcn-0 with transparent background, do not use dc__transparent since that might add padding